recipes/libpcsclite/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libpcsclite" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 856a1daed9a5ae1ac265c75ce9f81743e66a972e5be5a1d3a2196ed23f75ea26
+
+build:
+  skip: True  # [win]
+  script:
+    - ./bootstrap
+    - ./configure  # [linux]
+    - ./configure --disable-libsystemd  # [osx]
```

Comment:
According to the official website, we should not build on macOS:
> Apple reimplemented PC/SC above their own CryptoTokenKit API.
> You should not need to build pcsc-lite on macOS.

https://pcsclite.apdu.fr/#supported

In fact that's what I see too with `openconnect` (I am the maintainer of openconnect-feedstock): On macOS, the Yubikey support (which would rely on pcsc on Linux) is built and enabled by default, without any effort.

Reply:
Got it, do you know if Win build is impossible/inappropriate? I sort of assumed based on what was required

Reply:
I don't think this works on Windows. Even if it does, we can create a windows build later.